nav2_smac_planner/src/costmap_downsampler.cpp

```diff
@@ -126,7 +128,7 @@ void CostmapDownsampler::setCostOfCell(
   const unsigned int & new_my)
 {
   unsigned int mx, my;
-  unsigned char cost = 0;
+  unsigned char cost = _use_min_cost_neighbor ? 255 : 0;
```

Comment:
This is not a default value, it's a starting value of finding a minimum/maximum of the cells being downsampled and needs to be the lowest/highest `unsigned char` value so that `std::max`/`std::min` takes effect

Reply:
Good point, thanks